docs/user/howto/shared-deps.md

```diff
@@ -0,0 +1,165 @@
+# Working with shared dependencies
+
+Hypper charts understand the concept of shared dependency charts.
+
+A chart declared as a shared dependency is a chart that more than one chart may
+depend on. Charts deployed as shared dependencies are the analogous of system
+libraries in an OS: dependencies that are used by several consumers. One
+deployment may satisfy the shared dependencies of several charts.
+
+## Creating a Hypper chart with shared dependencies
+
+Let's create the simplest chart possible, and add charts as shared dependencies
+to it.
+
+First, we create a simple empty chart:
+
+```terminal
+$ helm create our-app
```

Comment:
(This is not a blocker) We should port that function over to hypper.

Reply:
Sounds like a good idea. But even with that, I would like to keep the how-to so people create a hypper chart from a helm chart (instead of calling a command), to know the differences better.

Reply:
You are right, we should add two sub-chapters later on
- creating a chart with shared dependencies from a helm chart
- creating a hypper chart2